channels/channel.videoland/videolandnl/chn_videolandnl.py

```diff
@@ -255,6 +257,34 @@ def extract_program_id(self, data: str) -> Tuple[Union[str, JsonHelper], List[Me
         self.__program_id = json_data.get_value("entity", "id", fallback=None)
         return json_data, []
 
+    def postprocess_episodes(self, data, items: list[MediaItem]) -> list[MediaItem]:        
```

Comment:
It needs to be `List[MediaItem]` with a capital `L`. Same goes for the response type.

Reply:
Thank you! Would this fix the failing tests on 3.8? I will probably push a new commit later today.